### PR TITLE
ADR-806 Declare Total Spirits screen

### PR DESCRIPTION
**Pull request by sini-george**

### CODE REVIEW

**Review by Ewan-James-Donovan — APPROVED** [review]
Lgtm!

File: app/views/DeclareSpiritsTotalView.scala.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form))
+        }
+
+        @sectionHeading(
+            id = "declare-spirits-total-section",
+            text = messages("section.alcoholDutyQuarterlySpiritsReturn"),
+        )
+
+        @pageHeading(messages("declareSpiritsTotal.heading"))
```
Comment (yasaracar) [inline app/views/DeclareSpiritsTotalView.scala.html:46]: You don't need to split pageHeading if there is no separate content like a paragrarh between the heading and the radio buttons.

Comment (Ewan-James-Donovan) [reply app/views/DeclareSpiritsTotalView.scala.html:46]: True:

 @govukInput(
        InputViewModel(
            field = form("declare-spirits-total-input"),
            label = LabelViewModel(messages("declareSpiritsTotal.heading")).asPageHeading(),

            ....

**Review by Ewan-James-Donovan — CHANGES_REQUESTED** [review]

**Review by Ewan-James-Donovan — APPROVED** [review]
LGTM

**Review by yasaracar — APPROVED** [review]

**Pull request merged**